SYCL/Basic/zero_size_range.cpp

```diff
@@ -1,4 +1,5 @@
 // RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: %t.out
```

Comment:
This line will execute the test only on device/bakcend which is chosen by default, for example gpu with l0 backend.
I guess you need to use placeholders to correctly run this test on all devices of interest. Please take a look at other tests, for example: https://github.com/intel/llvm-test-suite/blob/intel/SYCL/Basic/buffer/buffer_full_copy.cpp